index.html

```diff
@@ -344,6 +392,5 @@ <h2 class="contact">Contact me</h2>
         <button type="submit" class="btn btn-width">Get In Touch</button>
```

Comment:
Kindly note that the submit button of your contact form is a little misplaced in the mobile version. Please fix that too 🤗

Your design:
![image](https://user-images.githubusercontent.com/73714615/186908423-949dabf7-5d3a-49ea-9cb7-eb91290267af.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186908477-cec020b9-97ab-45a4-8ef1-66cf9c6b0be5.png)